Update README for release (#44)
virtrolio/virtrolio-site: README.md

<img align="left" src="https://github.com/virtrolio/virtrolio-site/blob/master/src/assets/images/logo_reg.png" width="200px" height="200px">

# Virtrolio

> Virtrolio is a webservice that lets you send messages to a friend or colleague's virtual book. It was originally created due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Link to the website: https://virtrolio.web.app/

Reviewer: ```suggestion
Virtrolio was founded due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Virtrolio is a web service that lets you privately send messages to a friend or colleague and allows only them to view the messages that are destined for them. With the help of some HTML, CSS, Typescript, love, sweat, and tears, we've combined the fun, privacy, and intimacy of yearbook signing with online features such as image embedding and more! 

Check it out! All you need is a Google Account: https://virtrolio.web.app/
```

Reviewer 2: The link to the website should be at the very top, before the first section.

Author: I think it makes more sense in context if it comes afterwards

> Virtrolio is a webservice that lets you send messages to a friend or colleague's virtual book. It was originally created due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Link to the website: https://virtrolio.web.app/

Reviewer 3: ```suggestion
Virtrolio is a webservice that lets you send messages to a friend' or colleague's virtual yearbook. It was originally created due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Link to the website: https://virtrolio.web.app/
```

<br>
<br>

## How We Built It

> Virtrolio is built using Angular and Bootstrap in the frontend, with Firebase for data storage, hosting, and user authentication. Firebase was chosen for it's ease of use and integration with various login systems. By using well-tested login systems we remove the need to worry about login security.

Reviewer: ```suggestion
Virtrolio is built using Angular and Bootstrap in the front end, with Firebase for data storage, hosting, and user authentication. Firebase was chosen for its ease of use and integration of various secure login authentication providers.
```

## Our Goal

> The goal of virtrolio is to allow users to write memorable messages to one another. We hope to give everyone the chance to do this, even if they cannot physically send anything.

Reviewer: ```suggestion
The goal of virtrolio is to allow users to write memorable messages to one another. We hope to give everyone the chance to do this, even if they cannot physically be together.
```
